schemas/context/advertising.example.1.json

```diff
@@ -18,6 +18,13 @@
       "xdm:value": 1.0
     }
   },
+  "xdm:adAssetReference": {
+    "@id": "https://data.adobe.io/entities/ad-id/AD1",
```

Comment:
Is this really an AdID (https://www.ad-id.org/)?  If so, we need a reference to it.  If not, then someone needs to give it a new name...

Reply:
Referenced AdID in the description.